configs/gooddata.json

```diff
@@ -2,11 +2,14 @@
   "index_name": "gooddata",
   "start_urls": [
     "https://sdk.gooddata.com/gooddata-ui/docs/",
-    "https://sdk.gooddata.com/gooddata-ui/docs/about_gooddataui.html"
+    "https://sdk.gooddata.com/gooddata-ui/docs/about_gooddataui.html",
+    "https://sdk.gooddata.com/gooddata-ui-apidocs/*/docs/index.html"
+
   ],
   "stop_urls": [],
   "sitemap_urls": [
-    "https://sdk.gooddata.com/gooddata-ui/sitemap.xml"
+    "https://sdk.gooddata.com/gooddata-ui/sitemap.xml",
+    "https://sdk.gooddata.com/gooddata-ui-apidocs/*/sitemap.xml"
```

Comment:
Unfortunately, regexes are not supported in the `sitemap_urls`, so you'd have to define all of them manually, or have a sitemap that groups them all!